### DISCUSSION TITLE
Get rid of the daily streak or have the option to disable them

### DISCUSSION THREAD

**Discussion by FDelporte** [dailydotdev/daily#1613] [Feature Requests]
I love daily.dev but get crazy of the "let's make a game of this". I don't need this and it bugs me enough to delete it soon from my browser... Please let me disable this!

**Comment by rebelchris** [comment]
Hi @FDelporte ,

You can actually disable it fully here:
![Screenshot 2024-11-22 at 10 50 00](https://github.com/user-attachments/assets/fbd3a9a0-10f8-4f79-acdf-8d844115da2b)

Read more here as well:
https://docs.daily.dev/docs/your-profile/weekly-goal#enablingdisabling-reading-streak

**Reply by FDelporte** [reply]
Thanks, indeed. I found it after posting it here... It's obviously not clear enough... It should be in the warning box that you can disable it. Seems to annoy others too: https://github.com/dailydotdev/daily/discussions/1143

**Reply by rebelchris** [reply]
Thanks for raising the concern, I've let our team know about it again.
